docs.yml

```diff
@@ -0,0 +1,3 @@
+title: Fluri
```

Comment:
I think my preference is to match the library name.  @evanweible-wf @maxwellpeterson-wf @jayudey-wf , what are your preferences?